frontend/src/components/opportunity/OpportunityIntro.tsx

```diff
@@ -54,16 +54,14 @@ const OpportunityIntro = ({ opportunityData }: Props) => {
 
   return (
     <>
-      <div className="usa-prose">
-        <p className="usa-intro line-height-sans-5 tablet-lg:font-sans-lg margin-top-0">{`${t("agency")} ${agencyName}`}</p>
-        <AssistanceListingsDisplay
-          assistanceListings={opportunityData.opportunity_assistance_listings}
-          assistanceListingsText={t("assistance_listings")}
-        />
-        <p className="tablet-lg:font-sans-2xs">
-          {lastUpdated(opportunityData.updated_at)}
-        </p>
-      </div>
+      <p className="usa-intro line-height-sans-5 tablet-lg:font-sans-lg margin-top-0">{`${t("agency")} ${agencyName}`}</p>
+      <AssistanceListingsDisplay
+        assistanceListings={opportunityData.opportunity_assistance_listings}
+        assistanceListingsText={t("assistance_listings")}
+      />
+      <p className="tablet-lg:font-sans-2xs">
+        {lastUpdated(opportunityData.updated_at)}
+      </p>
```

Comment:
- Don't need `.usa-prose` anymore (or this container `div`)
- Not sure about the paragraph size, will confirm with @crystabelrangel and follow up w/ another PR if this is wrong